code/modules/admin/admin.dm

```diff
@@ -1292,6 +1302,8 @@ var/global/floorIsLava = 0
 		if(!chosen)
 			return
 
+	_preloader = new(varchanges, chosen)
```

Comment:
Uh, you're not doing anything with this, are you?

Reply:
Preloader is always deleted after processing one atom, so there won't be any problem with that
I'll clarify that in a comment though

Reply:
oh you probably meant something else

_preloader is hooked to atom/New(), so there's no need to call anything after initializing it